### ISSUE TITLE
2.0 checklist/braindump

### ISSUE THREAD

**Issue by zachdaniel** [ash-project/ash#290]
They don't all have to happen all at once.

# Ash Core
- [ ] Go through all the tickets, have them organized (some of them cancelled, e.g I don't think we should rename destroy -> delete, even though it was voted overwhelmingly in favor of), placed on a roadmap, and anything pre-2.0.0 completed.
- [ ] `manage_relationship(...)` and `argument ...` combined into a single `manage_relationship(:relationship_name, :argument_name, opts)` call. will require updating all the extensions that look for managed relationships
- [ ] manual read actions for resources w/ data layers
- [ ] (maybe optional) I was hoping to have 2.0.0 come out with a new website and docs, the stuff discussed in docs-overhaul. But that doesn't necessarily have to happen before 2.0.0 comes out.
- [ ] option to set process-context for actor, context, and tenant.

# AshPostgres
- [ ] some kind of "here be dragons" escape hatch to modify the data layer query. This can actually likely be generalized, since all data layers have a custom query object
- [ ] a non-release-candidate ecto release to depend on (right now master is pointed at ecto master, which just won't do)

# AshGraphql
- [ ] some kind of complexity/query-size limiting feature-set
- [ ] to use the new managed relationship stuff (once it exists) 
- [ ] more tests

# AshPhoenix 
doesn't need a 1.0 for a while, but
- [ ] use the new managed relationship stuff

# AshJsonApi
also isn't ready, but
- [ ] use the new managed relationship stuff
- [ ] complete the json schema feature set, needs more tests
- [ ] some kind of complexity/query-size limiting feature-set

**Comment by zachdaniel** [comment]
This isn't really useful as an open issue.
